charts/gardener-extension-shoot-rsyslog-relp/templates/clusterrole.yaml

```diff
@@ -54,25 +85,35 @@ rules:
   - extensions.gardener.cloud
   resources:
   - extensions
-  - extensions/status
   verbs:
-  - get
   - list
   - watch
+- apiGroups:
+  - extensions.gardener.cloud
+  resources:
+  - extensions
+  - extensions/status
+  resourceNames:
+  - shoot-rsyslog-relp
+  verbs:
+  - get
   - patch
   - update
 - apiGroups:
   - ""
   resources:
   - secrets
```

Comment:
Doesn't the extension library need list for secrets anywhere (when cleaning up ManagedResource secrets)?

Reply:
Doesn't seem to be the case. The secrets are deleted directly by name.

Reply:
OK, thanks for explaining. I hope that's the case then.